pages/survey-form/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta sharset="utf-8">
+    <meta name="viewport" content="device-width", initial-scale 1.0>
+    <title>SURVEY-FORM</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="icon" href="icon.ico">
+  </head>
+
+  <body>
+
+    <div class="container">
+
+      <header>
+        <h1 class="center-text">freeCodeCamp Survey Form</h1>
+        <p class="header-description center-text">Thank you for taking the time to help us improve the platform</p>
+      </header>
+
+      <form class="form">
+
+        <div class="camp">
+          <label for="name">Enter Your Name: </label>
+          <input id="name" placeholder="Christopher" class="camp-input" required />
+        </div>
+
+        <div class="camp">
```

Comment:
It seems like `camp` is a random class name. Shouldn't we use a more descriptive name? `form-section` could be one.